packages/autorest.python/autorest/codegen/serializers/model_serializer.py

```diff
@@ -67,14 +69,14 @@ def super_call(self, model: ModelType):
         return f"super().__init__({self.properties_to_pass_to_super(model)})"
 
     @staticmethod
-    def initialize_discriminator_property(model: ModelType, prop: Property) -> str:
+    def initialize_discriminator_property(
```

Comment:
do we still need a special case for initializing discriminator properties? I believe it should be the same as initializing a constant